pymtl3/dsl/ComponentLevel2.py

```diff
@@ -451,7 +464,7 @@ def add_constraints( s, *args ): # add RD-U/WR-U constraints
           s._dsl.WR_U_constraints[ x0.var ].add( (sign, x1.func) )
 
   #-----------------------------------------------------------------------
-  # elaborate
+  # elaborate-related public APIs
```

Comment:
Did you mean just `API` instead of `public API`?

Reply:
I guess I copy-paste a lot and now I need to fix them ...